scripts/deploy.ts

```diff
@@ -160,13 +158,29 @@ async function main() {
   console.log(`${contractName} deployed and verified on Etherscan!`)
   console.log('Contract address:', address)
   console.log('Etherscan URL:', getEtherscanUrl(chainName, chainId, address))
-}
 
-function parseError(error: Error | unknown) {
-  console.log(
-    'Error verifiying contract on Etherscan:',
-    error instanceof Error ? error.message : error
+  console.log('Import old founders accounts...')
+  const founders = await getCountAddressAddedToAllowMap(
+    '0x91002bd44b9620866693fd8e03438e69e01563ee',
+    provider
   )
+  await contract.legacyBatchMint(
+    founders,
+    founders.map(() => 2)
+  )
+
+  console.log('Import old vs accounts...')
+  const vc = await getCountAddressAddedToAllowMap(
+    '0xe8c7754340b9f0efe49dfe0f9a47f8f137f70477',
```

Comment:
Can we put this address into `const` with proper name? It's probably `const oldVcContract` or something